client/components/tinymce/plugins/insert-menu/menu-items.jsx

```diff
@@ -1,24 +1,41 @@
+/**
+ * External dependencies
+ */
 import React from 'react';
-
 import Gridicon from 'gridicons';
 import i18n from 'i18n-calypso';
 
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
 const GridiconButton = ( { icon, label } ) => (
 	<div>
 		<Gridicon className="wpcom-insert-menu__menu-icon" icon={ icon } />
 		<span className="wpcom-insert-menu__menu-label">{ label }</span>
 	</div>
 );
 
```

Comment:
Although at the moment it's not a big deal, it may be a good idea to add `/* eslint-enable wpcalypso/jsx-classname-namespace */` to wrap the GridiconButton.

Or, if we had the energy to adapt this to the new standards and renaming the class to `insert-menu__nemu-item` the Golden Goddesses of the CSS Calypso Triforce would be even happier!